machine.go

```diff
@@ -249,6 +249,18 @@ func (d *Driver) MachineGet(id string) (*MachineGetResponse, error) {
 	return response, nil
 }
 
+// MachineConsolePassword returns the consolepassword of the machine
+func (d *Driver) MachineConsolePassword(id, reason string) (*models.V1MachineConsolePasswordResponse, error) {
+	cp := machine.NewGetMachineConsolePasswordParams()
+	cp.Body = &models.V1MachineConsolePasswordRequest{
+		ID:     &id,
+		Reason: &reason,
+	}
+
+	resp, err := d.machine.GetMachineConsolePassword(cp, nil)
+	return resp.Payload, err
```

Comment:
Needs `if err != nil` to prevent nil pointer. We're getting close. :)

Reply:
done